app/controllers/tags_controller.rb

```diff
@@ -162,8 +164,8 @@ def list
       @list.more_topics_url = construct_url_with(:next, list_opts)
       @list.prev_topics_url = construct_url_with(:prev, list_opts)
       @rss = "tag"
-      @description_meta = I18n.t("rss_by_tag", tag: tag_params.join(" & "))
-      @title = @description_meta
+      @title = I18n.t("rss_by_tag", tag: tag_params.join(" & "))
+      @description_meta = @tag&.description.presence || @title
```

Comment:
```suggestion
      @description_meta = @tag.description&.presence || @title
```

I don't think `@tag` will ever be `nil` since `fetch_tag` raises an error if it is.

Reply:
`fetch_tag` returns early when the `tag_id` is "none", since that's a magic tag that doesn't really exist.